src/Kernel/Structures/_Utf8.php

```diff
@@ -32,6 +34,16 @@ public function getLength()
         return $this->length;
     }
 
+    /**
+     * @param bool $enable
+     * @return _Utf8
+     */
+    public function enableWrite(bool $enable): self
+    {
+        $this->isWritable = $enable;
+        return $this;
+    }
+
```

Comment:
Actually we noticed the implementation around `isWritable`. Does it make sense to get rid of it?

Reply:
I'm thinking that it should normally be read-only.